usaspending_api/etl/management/commands/load_historical.py

```diff
@@ -0,0 +1,85 @@
+from datetime import datetime
+from decimal import Decimal
+import logging
+import os
+import re
+
+from django.conf import settings
+from django.core.exceptions import ObjectDoesNotExist, MultipleObjectsReturned
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+from django.core.serializers.json import json
+from django.db import connections
+from django.db.models import F, Q
+from django.core.cache import caches
+import pandas as pd
+import numpy as np
+
+from usaspending_api.accounts.models import (
+    AppropriationAccountBalances, AppropriationAccountBalancesQuarterly,
+    TreasuryAppropriationAccount)
+from usaspending_api.awards.models import (
+    Award, FinancialAccountsByAwards,
+    TransactionAssistance, TransactionContract, Transaction)
+from usaspending_api.financial_activities.models import (
+    FinancialAccountsByProgramActivityObjectClass, TasProgramActivityObjectClassQuarterly)
+from usaspending_api.references.models import (
+    Agency, LegalEntity, Location, ObjectClass, RefCountryCode, Cfda, RefProgramActivity)
+from usaspending_api.submissions.models import SubmissionAttributes
+from usaspending_api.etl.award_helpers import (
+    get_award_financial_transaction, update_awards, update_contract_awards,
+    update_award_categories, get_awarding_agency)
+from usaspending_api.etl.helpers import get_fiscal_quarter, get_previous_submission
+from usaspending_api.etl.broker_etl_helpers import dictfetchall, PhonyCursor
+from usaspending_api.etl.subaward_etl import load_subawards
+from usaspending_api.references.helpers import canonicalize_location_dict
+
+from usaspending_api.etl.helpers import update_model_description_fields
+from usaspending_api.etl.management import load_base
```

Comment:
It looks like a lot of these import statements are coming from a copy/paste? Would be great to reduce the noise by removing anything not needed.